lightning/restore/file_checkpoints.proto

```diff
@@ -28,12 +28,14 @@ message TableCheckpointModel {
     uint32 status = 3;
     int64 alloc_base = 4;
     repeated EngineCheckpointModel engines = 6;
+    repeated uint32 index_engine_statuses = 7;
 }
 
 message EngineCheckpointModel {
     uint32 status = 1;
     // key is "$path:$offset"
     map<string, ChunkCheckpointModel> chunks = 2;
+    int32 engine_id = 3;
```

Comment:
You shouldn't need engine_id at all! In RFC 4 the status should be written into the `index_engine_statuses` field!

Reply:
Let's discuss about this tomorrow morning